tests/common/test_transaction.py

```diff
@@ -1023,7 +1018,6 @@ def test_create_transfer_transaction_multiple_io(user_pub, user_priv,
                     'owners_before': [
                         user2_pub
                     ],
-                    'fid': 1,
                     'fulfillment': None,
                     'input': {
                         'txid': tx.id,
```

Comment:
Can you also design a test with multiple inputs/outputs and see if you can screw the sorting of either conditions or fulfillments?